anda/langs/go/u-root/golang-github-u-root.spec

```
@@ -28,12 +28,13 @@ file system (initramfs) containing a busybox-like set of tools written in Go.}
                         tools/golang_patched_dce/README.md
 
 Name:           %{goname}
-Release:        1%?dist
+Release:        2%?dist
 Summary:        A fully Go userland with Linux bootloaders! u-root can create a one-binary root file system (initramfs) containing a busybox-like set of tools written in Go
 
 License:        BSD-3-Clause
 URL:            %{gourl}
 Source:         %{gosource}
+Patch0:         https://github.com/u-root/u-root/compare/v0.14.0..c2602c67706500f0e81dbeb468377fffd22d884f.diff
```

i'd a bit further to where nixpkgs is since thats tested and knwon good
```suggestion
Patch0:         https://github.com/u-root/u-root/compare/v0.14.0..a620c4fc0eeeaa71ea68c27d6ef96352ed814829.diff
```